server/routes/createThread.ts

```diff
@@ -105,7 +105,9 @@ const createThread = async (models, req: Request, res: Response, next: NextFunct
       return next(err);
     }
   } else {
-    return next(Error('Must pass a topic_name string and/or a numeric topic_id'));
+    if ((community || chain).topics.length) {
```

Comment:
I generally find that `.length > 0` more clearly expresses what's happening when checking for length, although this could be something to discuss in #style